scores.csv

```diff
@@ -0,0 +1,15 @@
+Name,Score,Date
+Test1,49,12/09/1999
+Test2,65,03/07/2012
+Test3,23,21/05/2017
```

Comment:
Remove this file and make sure the game will create an empty scores file if it doesn't exist